modules/cli-administrator-cluster-management.adoc

```diff
@@ -5,6 +5,15 @@
 [id="cli-cluster-management-commands-{context}"]
 = Cluster management CLI commands
 
+== must-gather
+
+Gather debugging information.
```

Comment:
I just put in a [PR](https://github.com/openshift/openshift-docs/pull/14766) about the log gathering command to use during a failed installation. Since there are different log gathering commands for different stages of cluster existence, do you think it's appropriate to point out the difference?

Reply:
Yes

Reply:
@kalexand-rh Good question. Agree that this could definitely use a better description, I just pulled in what was submitted in the other PR for now. Ideally we wanted to add a section on using must-gather, but we aren't going to get to it by GA since I wasn't able to get the SME input that I needed in time.

@sferich888 Would this be an appropriate summary to use here for must-gather, or do you have a better suggestion?

```
Bulk collect data about the current state of your cluster to debug issues.
```

Reply:
Yes